### DISCUSSION TITLE
citations which are not useful/relevant to the query

### DISCUSSION THREAD

**Discussion by alfasin** [perplexityai/api-discussion#94] [Q&A]
The field `response.choices[0].message.content` returned a text that says that only **two** links were found relevant but the `citations` array contained **5** links.

Indeed only the first two were relevant, and I have no clue how were the other 3 chosen...

Is there a way to instruct the LLM to return only relevant citations?

**Comment by Mathias-RS** [comment]
Or is there a way to get more than 5 citations?
